### PR TITLE
Update internship application card

### PR DESCRIPTION
**Pull request by ongch00**
Remove details in InternshipApplicationCard that will be in another card
Add color coded status based on the colors used for statistics
Add the following color coding for cards:
    -Green if it is upcoming
    -Red if ghosted

### CODE REVIEW

File: src/main/java/seedu/address/ui/InternshipApplicationCard.java [hunk 1]
```diff
@@ -42,20 +50,49 @@
...
+            break;
+        case OFFERED:
+            statusLabel.setStyle("-fx-text-fill: black; -fx-background-color: #ffdd57");
+            break;
+        case REJECTED:
+            statusLabel.setStyle("-fx-background-color: #ff3860; ");
+            break;
+        case GHOSTED:
+            statusLabel.setStyle("-fx-background-color: SLATEGRAY");
+            break;
```
Comment (wxwxwxwx9) [inline src/main/java/seedu/address/ui/InternshipApplicationCard.java]: I think we should abstract out the magic value

File: src/main/resources/view/InternshipApplicationListCard.fxml [hunk 2]
```diff
@@ -27,12 +28,7 @@
                 <Label fx:id="company" text="\$first" styleClass="cell_big_label" />
             </HBox>
             <Label fx:id="role" styleClass="cell_small_label" text="\$role" />
-            <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-            <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-            <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
-            <Label fx:id="applicationDate" styleClass="cell_small_label" text="\$applicationDate" />
-            <Label fx:id="priority" styleClass="cell_small_label" text="\$priority" />
-            <Label fx:id="status" styleClass="cell_small_label" text="\$status" />
+            <FlowPane fx:id="status" alignment="CENTER_LEFT" />
```
Comment (wxwxwxwx9) [inline src/main/resources/view/InternshipApplicationListCard.fxml:31]: nice!

File: src/main/java/seedu/address/ui/InternshipApplicationCard.java (hunk 1 above)
Comment (ongch00) [reply src/main/java/seedu/address/ui/InternshipApplicationCard.java]: Ohh yea, good point

**Comment by foochifa** [comment]
Good job!

**Pull request merged**